pkg/util/helm/values.go

```diff
@@ -31,10 +31,10 @@ func FetchImageFromHelmValues(clientSet *kubernetes.Clientset, namespace, name s
 		KubeClient:       kube.New(restClientGetter),
 	})
 	getValues.AllValues = true
-	helmValues, err := getValues.Run("harvester")
+	helmValues, err := getValues.Run(name)
```

Comment:
@FrankYang0529 FYI, the given chart name is used, instead of fixed "Harvester", thanks.